### ISSUE TITLE
inifinite wait on load of region file

### ISSUE THREAD

**Issue by arusstaylor** [CARTAvis/carta-frontend#1361]
I am trying to load a region file that consists of ellipses for sources found with pybdsf.  First few lines of the file below. 
CARTA seems to hang when I press the load button.

There re 6228 ellipse lines in total in the file.  Is this too large for CARTA?

#CRTF 
global color=green, linewidth=1, coord=J2000 
ellipse[[  33.4673deg,   -4.6302deg], [  13.48arcsec,   10.02arcsec],  262.2deg] 
ellipse[[  33.4690deg,   -4.9074deg], [  10.30arcsec,    5.31arcsec],  177.7deg] 
ellipse[[  33.4709deg,   -5.3381deg], [   9.36arcsec,    5.75arcsec],  146.0deg] 
ellipse[[  33.4732deg,   -4.9719deg], [   9.04arcsec,    4.14arcsec],  156.2deg] 
ellipse[[  33.4738deg,   -4.3719deg], [   7.45arcsec,    5.96arcsec],  130.5deg] 
ellipse[[  33.4742deg,   -4.6058deg], [   6.79arcsec,    5.43arcsec],  167.5deg] 
ellipse[[  33.4778deg,   -5.2174deg], [   7.82arcsec,    5.50arcsec],  142.4deg] 
ellipse[[  33.4780deg,   -5.0185deg], [  10.88arcsec,    6.34arcsec],  155.9deg]

**Comment by veggiesaurus** [comment]
@arusstaylor I think this is a duplicate of #1252

**Comment by arusstaylor** [comment]
indeed
